File: Light it up/sol.cpp
// Online C++ compiler to run C++ program online
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n;
    cin >> n;
    vector<int> arr(n);
    for (int &e: arr) cin >> e;
    int ans = 0;
    for (int i = 0; i < n; i++) {
        if (arr[i] == 0 && (i == 0 || (i > 0 && arr[i - 1] == 1)))
        ans++;
    }
    cout << ans << '\n';
    

    return 0;
}

File: Light it up/sol.py
n = int(input())

if n:
    arr = list(map(int, input().split()))
ans = 0

for i in range(n):
    if arr[i] == 0 and (i == 0 or (i > 0 and arr[i - 1] == 1)):
        ans += 1

print(ans)